### DISCUSSION TITLE
When can legacy Discovery mode be supported?

### DISCUSSION THREAD

**Discussion by deanz-io** [rqlite/rqlite#977] [Q&A] [answered]
I am currently using version 6.6 and using the legacy Discovery mode. I hope I can still use the legacy Discovery mode if I upgrade to version 7.0 or later

**Comment by otoolep** [comment]
Can you tell me more about your use case? If I understand why legacy Discovery is useful to you I might be able to offer an alternative, or add support for it back into 7.0 as long as you are prepared to accept some small changes to how it works.

**Comment by otoolep** [comment]
Have you thought about trying out the new autoclustering feature, introduced in 7.1.0?

https://github.com/rqlite/rqlite/blob/master/DOC/AUTO_CLUSTERING.md#automatic-bootstrapping

**Comment by deanz-io** [comment]
I don't want to introduce etcd or consul just because I'm using rqlite, so far I've implemented the discovery center myself to implement clustering.
The new autoclustering feature needs to know the addresses of all nodes in advance, which is less convenient than the old clustering feature, Only the address of the discovery center is needed

**Comment by otoolep** [comment, marked as answer]
I will not be re-implementing legacy mode.
